src/ASM/ASMs2D.C

```diff
@@ -1297,44 +1297,40 @@ bool ASMs2D::updateCoords (const Vector& displ)
 }
 
 
-void ASMs2D::getBoundaryNodes (int lIndex, IntVec& nodes) const
+void ASMs2D::getBoundaryNodes (int lIndex, IntVec& nodes, int basis) const
 {
-  if (!surf) return; // silently ignore empty patches
+  if (basis == 0)
+    basis = 1;
+
+  if (!getBasis(basis)) return; // silently ignore empty patches
```

Comment:
Doxy er feil. Bryter feks mixed og fsi hvis alle blir returnert

-------- Opprinnelig melding --------
Fra: kmokstad <notifications@github.com>
Dato: 03.05.2016 22:13 (GMT+01:00)
Til: OPM/IFEM <IFEM@noreply.github.com>
Ko: Arne Morten Kvarving <Arne.Morten.Kvarving@sintef.no>, Author <author@noreply.github.com>
Emne: Re: [OPM/IFEM] PETSc reboot - general domain decomposition (#7)


In src/ASM/ASMs2D.C<https://github.com/OPM/IFEM/pull/7#discussion_r61947480>:

>  {
> -  if (!surf) return; // silently ignore empty patches
> +  if (basis == 0)
> +    basis = 1;
> +
> +  if (!getBasis(basis)) return; // silently ignore empty patches


vennligst bruk this-> så er det tydeligere at getBasis er en member og ikke en global static funksjon (konvensjon jeg har lagt meg på ellers). Dessuten står det over at basis==0 betyr for alle basiser. Hvordan forklares det da at den her resettes til 1? Da vil vel man aldri kunne få ut for alle basisene (om nå det skulle være ønskelig).
Eller er doxyen feil?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/OPM/IFEM/pull/7/files/c7217e143f115795decd5a0825082a79366a80f1..42876e3e4e0331f01b4f774f65f6bce431496644#r61947480>